.htaccess

```
@@ -142,9 +142,9 @@ DirectoryIndex index.php index.html index.htm
     RewriteCond %{REQUEST_FILENAME}\.gz -s
     RewriteRule ^(.*)\.js $1\.js\.gz [QSA]
 
-    # Serve correct content types, and prevent mod_deflate double gzip.
-    RewriteRule \.css\.gz$ - [T=text/css,E=no-gzip:1]
-    RewriteRule \.js\.gz$ - [T=text/javascript,E=no-gzip:1]
+    # Serve correct content types, and prevent double compression from mod_deflate (gzip) and mod_brotli (br).
```

This line is running over the 80 character wrapping limit, could we just match Drupal is using with the abbreviated:
```suggestion
    # Serve correct content types, and prevent double compression.
```

---

I think the proposed comment change is fine. If it becomes an issue of confusion, we can address it in more detail later.